help/destinations/catalog/personalization/pega-v2.md

```diff
@@ -0,0 +1,137 @@
+* **[!UICONTROL Client Secret]**: The OAuth 2 [!DNL client secret] that you generated in your [!DNL Pega Customer Decision Hub] instance.
+
+### Fill in destination details {#destination-details}
+
+After establishing the authentication connection to the [!DNL Pega Customer Decision Hub], provide the following information for the destination:
+
+![Image of the UI screen showing completed fields for the Pega CDH destination details](../../assets/catalog/personalization/pega/pega-connect-destination-v2.png)
+
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Pega CDH Host Name]**: The Pega Customer Decision Hub Host Name to which the profile gets exported as json data.
+*  **[!UICONTROL Application alias]**: The application alias is configured for the Customer Decision Hub.
+
+## Activate audiences to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>* To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+See [Activate audience data to streaming profile export destinations](../../ui/activate-streaming-profile-destinations.md) for instructions on activating audiences to this destination.
+
+### Destination attributes {#attributes}
+
+In the [[!UICONTROL Select attributes]](../../ui/activate-streaming-profile-destinations.md#select-attributes) step, Adobe recommends that you select a unique identifier from your [union schema](../../../profile/home.md#profile-fragments-and-union-schemas). Select the unique identifier and any other XDM fields that you want to export to the destination.
+
+### Mapping example: activating profile updates in [!DNL Pega Customer Decision Hub] {#mapping-example}
+
+Below is an example of correct identity mapping when exporting profiles to [!DNL Pega Customer Decision Hub].
+
+Selecting source fields:
+
+* Select an identifier (For example: CustomerID) as source identity that uniquely identifies a profile in Adobe Experience Platform and [!DNL Pega Customer Decision Hub].
```

Comment:
Select a source identity that uniquely identifies a profile in Adobe Experience Platform and [!DNL Pega Customer Decision Hub]. For example: `CustomerID`.

Reply:
updated